cacerts/layer.go

```diff
@@ -0,0 +1,56 @@
+package cacerts
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/buildpacks/libcnb"
+	"github.com/paketo-buildpacks/libpak"
+	"github.com/paketo-buildpacks/libpak/bard"
+)
+
+type Layer struct {
+	CertPaths         []string
+	LayerContributor  libpak.LayerContributor
+	GenerateHashLinks func(dir string, certPaths []string) error
+	Logger            bard.Logger
+}
+
+func NewLayer(paths []string) *Layer {
```

Comment:
Using `Layer` and `NewLayer` here is too generic.  Be descriptive with the types and constructor.